### PR TITLE
Graphics Inventory #3 Pull Request

### PR DESCRIPTION
**Pull request by imadejski**
RunInventoryGraphics function created along with test_inventory_graphics file created to test new init frees. Makefile also created. The draw inventory graphics function is simply an eyeball test; these are all things that will be included in our demo.

### CODE REVIEW

**Review by dhirpalshah — APPROVED** [review]
awesome! eyeball tests are nice.

File: graphics_sandbox/inventory_graphics/test_inventory_graphics.c [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    sd.statistics = stat;
+    sd.mode = NUMERICAL;
+
+    g.dimensions = &dd;
+    g.inventory = &td;
+    g.statistics = &sd;
+
+    return g;
+} 
+*/
```
Comment (philipchenaux) [inline graphics_sandbox/inventory_graphics/test_inventory_graphics.c]: Is there a reason why all the above tests are commented out?

File: graphics_sandbox/inventory_graphics/test_inventory_graphics.c [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+Test(slot, new)
```
Comment (philipchenaux) [inline graphics_sandbox/inventory_graphics/test_inventory_graphics.c:64]: Please add comments describing what you are testing.

**Review by philipchenaux — CHANGES_REQUESTED** [review]
A few stylistic comments that must be addressed.

File: graphics_sandbox/inventory_graphics/test_inventory_graphics.c (hunk 1 above)
Comment (ukanyanwu) [reply graphics_sandbox/inventory_graphics/test_inventory_graphics.c]: These were going to supplement player_inventory tests, but were temporarily commented out. These are now uncommented, and the player_inventory new/init/free tests area also re-added.

File: graphics_sandbox/inventory_graphics/test_inventory_graphics.c (hunk 2 above)
Comment (ukanyanwu) [reply graphics_sandbox/inventory_graphics/test_inventory_graphics.c:64]: Comments have been added to the slot tests, along with the player_inventory tests that were re-added.

**Review by philipchenaux — APPROVED** [review]
Looks good to merge, well done!

**Comment by philipchenaux** [comment]
Issue Score: Excellent

Comments:
Great work, but please make sure that you do a thorough code review and, in the future, make sure to look out for the kind of issues the senior developer pointed out.

**Pull request merged**